cloud-init-master.yaml

```diff
@@ -5,16 +5,19 @@ packages:
   - apt-transport-https
   - ca-certificates
   - curl
+  - gpg
   - jq
   - [kubelet, __KUBERNETES_VERSION__]
   - [kubeadm, __KUBERNETES_VERSION__]
   - [kubectl, __KUBERNETES_VERSION__]
 
+bootcmd:
+  - curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.29/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
```

Comment:
I will replace the version into `__KUBERNETES_VERSION__` to specify the version from Makeflie.